version/constants.go

```diff
@@ -97,18 +133,47 @@ var (
 	}
 	ApricotPhase5DefaultTime = time.Date(2020, time.December, 5, 5, 0, 0, 0, time.UTC)
 
-	SunrisePhase0Times       = map[uint32]time.Time{}
-	SunrisePhase0DefaultTime = time.Date(2022, time.May, 16, 8, 0, 0, 0, time.UTC)
+	ApricotPhasePre6Times = map[uint32]time.Time{
+		constants.MainnetID: time.Date(2022, time.September, 5, 1, 30, 0, 0, time.UTC),
+		constants.FujiID:    time.Date(2022, time.September, 6, 20, 0, 0, 0, time.UTC),
+
+		constants.CaminoID:     caminoEarliestTime,
+		constants.ColumbusID:   columbusEarliestTime,
+		constants.KopernikusID: kopernikusEarliestTime,
+	}
+
+	SunrisePhase0Times = map[uint32]time.Time{
+		constants.CaminoID:     caminoEarliestTime,
+		constants.ColumbusID:   columbusEarliestTime,
+		constants.KopernikusID: kopernikusEarliestTime,
+	}
 
 	ApricotPhase6Times = map[uint32]time.Time{
 		constants.MainnetID: time.Date(2022, time.September, 6, 20, 0, 0, 0, time.UTC),
 		constants.FujiID:    time.Date(2022, time.September, 6, 20, 0, 0, 0, time.UTC),
+
+		constants.CaminoID:     caminoEarliestTime,
+		constants.ColumbusID:   columbusEarliestTime,
+		constants.KopernikusID: kopernikusEarliestTime,
+	}
+
+	ApricotPhasePost6Times = map[uint32]time.Time{
+		constants.MainnetID: time.Date(2022, time.September, 7, 3, 0, 0, 0, time.UTC),
+		constants.FujiID:    time.Date(2022, time.September, 7, 6, 0, 0, 0, time.UTC),
+
+		constants.CaminoID:     caminoEarliestTime,
+		constants.ColumbusID:   columbusEarliestTime,
+		constants.KopernikusID: kopernikusEarliestTime,
 	}
 	ApricotPhase6DefaultTime = time.Date(2020, time.December, 5, 5, 0, 0, 0, time.UTC)
```

Comment:
is this intended, to leave defaulttime variables for avax releases but remove it for athens and berlin? btw in the original PR, There is no phasedefaulttime for all phases.

Reply:
yes, its intended. Phase-specific default times were removed by avax in later cortinas.